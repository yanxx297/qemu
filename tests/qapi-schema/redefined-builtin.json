# FIXME: we should reject types that duplicate builtin names
{ 'type': 'size', 'data': { 'myint': 'size' } }
