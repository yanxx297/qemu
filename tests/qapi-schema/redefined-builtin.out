[OrderedDict([('type', 'size'), ('data', OrderedDict([('myint', 'size')]))])]
[]
[OrderedDict([('type', 'size'), ('data', OrderedDict([('myint', 'size')]))])]
